README.md

```diff
@@ -200,6 +200,60 @@ published on Hackage, or part of Stackage snapshots, using Bazel.
 
 [hazel]: https://github.com/formationai/hazel
 
+## Troubleshooting
```

Comment:
We should merge the two troubleshooting sections. No need to have two separate ones. Use the same format as the troubleshooting section below this one.